src/LanguageServer.php

```diff
@@ -100,4 +120,42 @@ public function exit()
     {
         exit(0);
     }
+
+    /**
+     * Parses workspace files, one at a time.
+     *
+     * @param string $rootPath The rootPath of the workspace. Is null if no folder is open.
+     * @return void
+     */
+    private function indexProject(string $rootPath)
```

Comment:
I think you cannot pass `null` if you type hint it as `string`. It needs to be `string $rootPath = null` then.